### PR TITLE
Feature/aws summit 2023

### PR DESCRIPTION
**Pull request by yolanv**
### Thank you for your contribution! Please go through this small checklist to prepare your pull request to be reviewed as soon as possible :)
* [X] All authors were added to `_data/authors.yml` with a picture and an up-to-date bio
* [X] The blog post image is of ratio 500x293
  * Feel free to leave a comment if you don't know how to resize your image and someone will gladly help you out
* [X] The [**one sentence per line** method](https://raw.githubusercontent.com/brandon-rhodes/blog/master/texts/brandon/2012/one-sentence-per-line.rst) has been adopted
* [X] A spell check has been done
* [X] All links open in a new tab
  * Example: `[Amazing tech blog](https://ordina-jworks.github.io){:target="_blank" rel="noopener noreferrer"}`
* [X] The post's content properly resizes according to the window's size
  * Tip: Just shorten your window's horizontal length to easily test this
  * To have images properly resize use `class="image fit"`
  * To have YouTube videos properly resize, wrap the iframe into a `<div class="responsive-video">`
* [X] If a table of content is: check whether all links are working correctly

### Any specific remarks for the reviewers or anything you need help with?
\<enter text here>

### Checklist for the reviewers before merging
* [ ] Publishing date of the post has been correctly set
  * This can be done by renaming the filename or by setting the date in the header section of the post

### CODE REVIEW

File: _posts/2023-06-20-aws-summit-amsterdam-2023.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2023-06-20-aws-summit-amsterdam-2023/andy-warfield.png' | prepend: site.baseurl }}" alt="Andy Warfield" class="image fit" style="margin:0px auto; max-width:100%">
+_Andy Warfield - the newest features and services_
+{: refdef}
+
+Andy also talked about Artificial Intelligence and Machine Learning, which is the newest hype and trend in IT.
+Amazon Bedrock, which is presented by Andy, allows you to create generative AI applications by using foundation models.
+Their main mission, as always, is that clients should be able to focus only on innovation and their applications, and the purpose of them, and not the surrounding infrastructure.
+
+With these kinds of topics, users are always afraid of their data and the privacy that surrounds the whole concept.
```
Comment (pietervincken) [inline _posts/2023-06-20-aws-summit-amsterdam-2023.md:49]:
```suggestion
With these kinds of topics, privacy is often a concern.
Users want to know how their data is used and what measures are taken to keep it safe.
```

File: _posts/2023-06-20-aws-summit-amsterdam-2023.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+AWS continues to assure that the data of their users remains safe.
+They always design their newest features and services with privacy on the first place, and they will to continue to invest in ways to ensure that they live up to their own standards.
```
Comment (pietervincken) [inline _posts/2023-06-20-aws-summit-amsterdam-2023.md:51]:
```suggestion
They always design their newest features and services with privacy as a primary concern, and they will to continue to invest in ways to ensure that they live up to their own standards.
```

File: _posts/2023-06-20-aws-summit-amsterdam-2023.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+In addition to product announcements, the keynote featured customer success stories, highlighting how organizations from various sectors have leveraged AWS to achieve significant business outcomes.
+The speaker spoke of companies that had successfully migrated their infrastructure to the cloud.
```
Comment (pietervincken) [inline _posts/2023-06-20-aws-summit-amsterdam-2023.md:54]:
```suggestion
They spoke of companies that had successfully migrated their infrastructure to the cloud.
```

File: _posts/2023-06-20-aws-summit-amsterdam-2023.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+There were a few companies that were there to share their experience.
```
Comment (pietervincken) [inline _posts/2023-06-20-aws-summit-amsterdam-2023.md:56]:
```suggestion
Some of these companies shared their experience on stage.
```

File: _posts/2023-06-20-aws-summit-amsterdam-2023.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+One of the best examples was the Efteling (an amusement park in The Netherlands), which was presented by Jonas Rietbergen.
+They added a functionality to their application where they estimate how occupied the park was at the current moment.
```
Comment (pietervincken) [inline _posts/2023-06-20-aws-summit-amsterdam-2023.md:58]:
```suggestion
They added a functionality to their application where they estimate how occupied the park at any point in time.
```

File: _posts/2023-06-20-aws-summit-amsterdam-2023.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+Thanks to these estimations, they were able to predict waiting times and recommend other alternatives in case the current ride was too tied up.
+Efteling did this by creating a data lake, integrating it with Sagemaker Model and saving predictions every 30 minutes to Aurora.
+These predictions could then be fetched by Lambda and exposed through API Gateway.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2023-06-20-aws-summit-amsterdam-2023/efteling.png' | prepend: site.baseurl }}" alt="Efteling" class="image fit" style="margin:0px auto; max-width:100%">
+_Efteling - predictions explained by Jonas Rietbergen_
+{: refdef}
+
+Other companies were HEMA, PostNL, Epic Games, which were presented by Andy and how these companies used the newest services and features.
```
Comment (pietervincken) [inline _posts/2023-06-20-aws-summit-amsterdam-2023.md:68]: Name some services here?

File: _posts/2023-06-20-aws-summit-amsterdam-2023.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+Takeaway (Just Eat) also gave a small presentation on how they use AWS to improve customer experience in their app.
```
Comment (pietervincken) [inline _posts/2023-06-20-aws-summit-amsterdam-2023.md:69]: Again, name 1 or 2 services and how it improved their situation.

**Review by pietervincken — CHANGES_REQUESTED** [review]

**Review by DuncanCasteleyn — APPROVED** [review]
Nice post

**Review by pietervincken — APPROVED** [review]

**Pull request merged**